tools/system_libs.py

```diff
@@ -2369,6 +2369,9 @@ def add_sanitizer_libs():
     add_library('libc')
     if settings.MALLOC == 'mimalloc':
       add_library('libmimalloc')
+      if settings.USE_ASAN:
+        # See https://github.com/emscripten-core/emscripten/issues/23288#issuecomment-2571648258
+        shared.exit_with_error('mimalloc should not be used with asan')
```

Comment:
`mimalloc is not compatible with with asan`?

Or "not currently" maybe?

Reply:
It sounded from the mimalloc people's responses there that this was by design (ASan uses its own allocator), IIUC.

Reply:
Sure, then 'mimalloc is not compatible with with asan' maybe?

Reply:
Done.